src/languages/php.md

```diff
@@ -83,7 +83,7 @@ This is the complete list of extensions that can be enabled:
 | intl         | *   | *   | *   | *   |
 | json         |     |     | *   | *   |
 | ldap         | *   | *   | *   | *   |
-| mcrypt       | *   | *   | *   | *   |
+| mcrypt       | *   | *   | *   |     |
```

Comment:
mcrypt is still available in PHP 7.0, just not enabled by default, so it should still be in the table.  Just remove this hunk.